pom.xml

```diff
@@ -23,12 +23,12 @@
   <parent>
     <groupId>org.hawkular</groupId>
     <artifactId>hawkular-parent</artifactId>
-    <version>43</version>
+    <version>46</version>
     <relativePath/>
   </parent>
   <groupId>org.hawkular.client</groupId>
   <artifactId>hawkular-java-client</artifactId>
-  <version>0.0.2-SNAPSHOT</version>
+  <version>0.0.16-SNAPSHOT</version>
```

Comment:
Ok, why not. My opinion is that it will be difficult to keep their version synced over time, since they're changing at different pace. But it's just an opinion :)